### PR TITLE
Charging the clockwork slab now uses overlays

### PR DESCRIPTION
**Pull request by ghost**
This includes stuff like Geis, Sentinel's Compromise, and Volt Blaster.

### CODE REVIEW

**Comment by ChangelingRain** [comment]
Since I overlayed these on the slab sprite, please make sure your before and after are exactly the same.

File: code/game/gamemodes/clock_cult/clock_scripture.dm [hunk 1]
```diff
@@ -271,7 +271,8 @@ Judgement: 12 servants, 5 caches, 300 CV, and any existing AIs are converted or
 	return ..()
 
 /datum/clockwork_scripture/ranged_ability/scripture_effects()
-	slab.icon_state = slab_icon
```
Comment (ChangelingRain) [inline code/game/gamemodes/clock_cult/clock_scripture.dm]: This means Geis and other things with inhands won't show up in-hand.

**Comment by Xhuis** [comment]
Due to how you did the overlays, getting everything to the right appearance should be fairly easy, assuming I can figure out how to get the correct colors. Comparing the pixel on the slab without the overlay and with the overlay is something I don't know how to do, though, so it might be some time.

**Comment by Xhuis** [comment]
After much strife, it is now accurate to the pixel.

**Comment by Xhuis** [comment]
actions.dmi pains my soul

**Comment by ChangelingRain** [comment]
I'm not sure why you're editing actions.dmi anyway.

**Comment by Xhuis** [comment]
Because I was a butt. There was a duplicate sprite that tilted me an unreasonably high amount and I tried to remove it while I was around, to little good.

**Comment by ChangelingRain** [comment]
Please do not do undocumented changes to unrelated files.

**Comment by Xhuis** [comment]
Well, it's gone now, and I'm making a different PR for it, so all's well, eh?

File: code/game/gamemodes/clock_cult/clock_scripture.dm [hunk 2]
```diff
@@ -271,7 +271,9 @@ Judgement: 12 servants, 5 caches, 300 CV, and any existing AIs are converted or
...
+	if(slab_overlay)
+		slab.add_overlay(slab_overlay)
+		slab.item_state = slab_overlay
```
Comment (Xhuis) [inline code/game/gamemodes/clock_cult/clock_scripture.dm]: This has also been fixed, in case you weren't aware due to how Git handles line comments.

**Comment by Xhuis** [comment]
This is ready!

**Comment by ChangelingRain** [comment]
I'd kind of prefer that the inhands are also made into overlays.

**Comment by Xhuis** [comment]
How do you mean? Like, have a base slab inhand with the overlay applied to that? Considering that there _is_ no base slab inhand, and these are only used sporadically, it seems like a waste. I can do it if you really feel like it's necessary, but it seems like a needless hoop.

**Comment by ChangelingRain** [comment]
These base slab inhands? That I gave you? http://puu.sh/wMVtU/c8bfa3e0ed.dmi

**Comment by Xhuis** [comment]
They don't exist in the _code._ That file itself was lost by me, but having it here makes things slightly different. Because I know nothing I say is going to change your opinion on making me do work for something small I'll just do it.

**Comment by ChangelingRain** [comment]
If you'd rather abrasive then yes, you can have abrasive.
Do X.

**Comment by Xhuis** [comment]
I'm trying to _avoid_ being abrasive. I'm just saying that it's often not clear when you and other maintainers say "I'd kind of prefer X" or other things whether or not you want them. I'd be more than happy to do these things but it's rarely clear.

I'm not trying to step on toes. Please try to understand my position here. I don't want to be your enemy or anyone else's.

**Comment by Xhuis** [comment]
I'm doing some rewriting of stuff tonight, so I wouldn't expect this until, like, tomorrow morning or afternoon. I still intend to do it, though.

**Comment by Xhuis** [comment]
Thanks to a certain hunk of a coder who rewrote screwdriver coloring, there was a system in place; inhands now use overlays, too. (Yes, they look the same!)

**Pull request merged**